pkg/test/dockerutil/container.go

```diff
@@ -397,7 +402,7 @@ func (c *Container) CopyFiles(opts *RunOpts, target string, sources ...string) {
 		c.copyErr = fmt.Errorf("ioutil.TempDir failed: %v", err)
 		return
 	}
-	c.cleanups = append(c.cleanups, func() { os.RemoveAll(dir) })
+	//c.cleanups = append(c.cleanups, func() { os.RemoveAll(dir) })
```

Comment:
Why do we remove cleanup here?

Reply:
Oops, come of this was for when I was debugging. Fixed now.